typescript/sdk/src/providers/explorerHealthTest.ts

```diff
@@ -0,0 +1,63 @@
+import { ChainMetadata } from '@hyperlane-xyz/sdk';
```

Comment:
Not new code. This was initially in the SDK, then moved to the registry because it was the only consumer, but now the widgets will need it to so I've moved it back 😅